atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueServiceSerializableTransactionTest.java

```diff
@@ -26,19 +26,19 @@
 
     @Override
     protected KeyValueService getKeyValueService() {
-        return CQLKeyValueService.create(
-                ImmutableSet.of("localhost"),
-                9160,
-                20,
-                1000,
-                "atlasdb",
-                false,
-                1,
-                1000,
-                10000000,
-                1000,
-                true,
-                false);
+        return CQLKeyValueService.create(ImmutableCassandraKeyValueServiceConfig.builder()
+                .addServers("localhost")
+                .port(9160)
+                .poolSize(20)
+                .keyspace("atlasdb")
+                .ssl(false)
+                .replicationFactor(1)
+                .mutationBatchCount(10000)
+                .mutationBatchSizeBytes(10000000)
```

Comment:
Cassandra 2.2 default max batch size is 50KB, so this could cause problems on a default configured C*